src/navigator_data_ingest/base/updated_document_actions.py

```diff
@@ -160,11 +160,13 @@ def update_dont_parse(
                 "s3://", update_config.pipeline_bucket, update_config.embeddings_input
             )
         ),
-        S3Path(
-            os.path.join(
-                "s3://", update_config.pipeline_bucket, update_config.indexer_input
-            )
-        ),
+        # Do not update the indexer input file - archiving the json is required
```

Comment:
Why don't we want to remove these lines?

Reply:
because I'm making changes to Mark's project & I thought it was useful to document. We'll probably end up removing them soon anyway...